### ISSUE TITLE
NTR: alpha-ketoglutarate high in CSF (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8642]
New term request alpha-ketoglutarate high in CSF (elthat_190712111633): 2-ketoglutarate; cerebrospinal fluid; 2-oxoglutarate; oxoglutaric; ketoglutaric  UMLS: 5195105

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000468
- label: Elevated CSF alpha-ketoglutarate concentration
- parents: [Abnormal CSF carboxylic acid concentration (HP:0500183)](https://hpo.jax.org/app/browse/term/HP:0500183)
- pmid: PMID:30635937
- def: The concentration of alpha-ketoglutarate in the cerebrospinal fluid (CSF) is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8642
- synonyms:
